<launch>
  <arg name="current_max" default="1000"/>
  <node name="mightex_device_controller" pkg="mightex_device" type="mightex_device_controller.py">
    <param name="current_max" value="$(arg current_max)"/>
  </node>
</launch>
